docs/source/locale/pt_BR/LC_MESSAGES/pipeline.po

```diff
@@ -32,3 +32,205 @@ msgstr "Este módulo permite que você configure a pipeline do seu modelo."
+"variable NEOMARIL_PASSWORD to set this"
+
+#: neomaril_codex.pipeline.NeomarilPipeline:9 of
+msgid "group"
+msgstr "group"
+
+#: neomaril_codex.pipeline.NeomarilPipeline:10 of
+msgid "Group the model is inserted"
+msgstr "Group the model is inserted"
+
+#: neomaril_codex.pipeline.NeomarilPipeline:11 of
+msgid "url"
+msgstr "url"
+
+#: neomaril_codex.pipeline.NeomarilPipeline:12 of
+msgid ""
+"URL to Neomaril Server. Default value is "
+"https://neomaril.staging.datarisk.net, use it to test your deployment "
+"first before changing to production. You can also use the env variable "
+"NEOMARIL_URL to set this"
+msgstr ""
+"URL to Neomaril Server. Default value is "
+"https://neomaril.staging.datarisk.net, use it to test your deployment "
+"first before changing to production. You can also use the env variable "
+"NEOMARIL_URL to set this"
+
+#: neomaril_codex.pipeline.NeomarilPipeline:14 of
+msgid "python_version"
+msgstr "python_version"
+
+#: neomaril_codex.pipeline.NeomarilPipeline:14 of
+msgid ""
+"Python version for the model environment. Avaliable versions are 3.7, "
+"3.8, 3.9, 3.10. Defaults to '3.9'"
+msgstr ""
+"Python version for the model environment. Avaliable versions are 3.7, "
+"3.8, 3.9, 3.10. Defaults to '3.9'"
+
+#: neomaril_codex.pipeline.NeomarilPipeline:17
+#: neomaril_codex.pipeline.NeomarilPipeline.from_config_file:12
+#: neomaril_codex.pipeline.NeomarilPipeline.register_monitoring_config:7
+#: neomaril_codex.pipeline.NeomarilPipeline.run_deploy:12
+#: neomaril_codex.pipeline.NeomarilPipeline.run_monitoring:9
+#: neomaril_codex.pipeline.NeomarilPipeline.run_training:9
+#: neomaril_codex.pipeline.NeomarilPipeline.start:6 of
+msgid "Example"
+msgstr "Example"
+
+#: neomaril_codex.pipeline.NeomarilPipeline.from_config_file:1 of
+msgid "Load the configuration files for orchestrate the model"
+msgstr "Load the configuration files for orchestrate the model"
+
+#: neomaril_codex.pipeline.NeomarilPipeline.from_config_file
+#: neomaril_codex.pipeline.NeomarilPipeline.register_deploy_config
+#: neomaril_codex.pipeline.NeomarilPipeline.register_monitoring_config
+#: neomaril_codex.pipeline.NeomarilPipeline.register_train_config
+#: neomaril_codex.pipeline.NeomarilPipeline.run_deploy
+#: neomaril_codex.pipeline.NeomarilPipeline.run_monitoring of
+msgid "Parameters"
+msgstr "Parameters"
+
+#: neomaril_codex.pipeline.NeomarilPipeline.from_config_file:3 of
+msgid "Path of the configuration file, but it could be a dict"
+msgstr "Path of the configuration file, but it could be a dict"
+
+#: neomaril_codex.pipeline.NeomarilPipeline.from_config_file
+#: neomaril_codex.pipeline.NeomarilPipeline.run_deploy
+#: neomaril_codex.pipeline.NeomarilPipeline.run_training
+#: neomaril_codex.pipeline.NeomarilPipeline.start of
+msgid "Raises"
+msgstr "Raises"
+
+#: neomaril_codex.pipeline.NeomarilPipeline.from_config_file:6 of
+msgid "Undefined credentials"
+msgstr "Undefined credentials"
```

Comment:
Will handle the translation later, when tackling this issue -> https://linear.app/datarisk/issue/NEODV-717/[techdebt]-atualizar-as-traducoes-pt-br-do-codex